docs/extensions/hooks.md

```diff
@@ -207,6 +211,8 @@ export default ({ embed }, { env }) => {
 | `(<collection>.)items.update`  | The updated item                     | `keys`, `collection`                        |
 | `(<collection>.)items.promote` | The promoted item                    | `collection`, `item`, `version`             |
 | `(<collection>.)items.delete`  | The keys of the item                 | `collection`                                |
+| `<system-collection>.query`    | The items query                      | `collection`                                |
+| `<system-collection>.read`     | The read item                        | `query`, `collection`                       |
```

Comment:
I believe there was a reason why we didn't document this specific event in the past, but I can't remember the exact details (cc @rijkvanzanten).

Reply:
I dont think these events are implemented on all `<system-collections>` so this doc would have to be more specific on which collections are actually supported.

Reply:
I think they everything except for fields implements it I could be wrong here do.

Reply:
My guess would be at least the 3 that dont conform to the API: `collections`, `fields`, `relations` (maybe extensions too as it doesnt look like it extends the ItemsService)

Reply:
There is already an exception specified for `files` in the "System Collections" tip box below. We could do that for the other ones as well, but that might get confusing pretty quickly.

Reply:
We were hesitant with things like filter hooks on reads as it's a big footgun: You can easily brick the app/api by modifying the expected read output, especially for system collections. People will find a way though, so as long as we're obvious that it's a bit at-your-own-risk we should be transparent 👍🏻

Reply:
> There is already an exception specified for `files` in the "System Collections" tip box below. We could do that for the other ones as well, but that might get confusing pretty quickly.

I think the best option maby would be System Collections a table like

| system-collection | Events                                        |
| ----------------- | --------------------------------------------- |
| permissions       | `query`, `read`, `create`, `update`, `delete` |

Then my secondary question becomes what would be the best place to add the warning for <system-collection>.read
maby we should add <sup>[4]</sup> with it and link sup [4] to a warning?

Reply:
@nickrum Any comments on the message above if not please add a reaction and I will do the rest of the work today or tomorrow.

Reply:
Adding the exceptions to the list of system collections should be fine for now. Ideally all system collections would support all events, so we wouldn't need the exceptions.

We already have a warning box regarding performance when listening to read events further up. A similar box that warns about altering the payload of system collections below this table should be good for now, I think 🤔

Reply:
> My guess would be at least the 3 that dont conform to the API: `collections`, `fields`, `relations` (maybe `extensions` too as it doesnt look like it extends the ItemsService)

extensions is not even in the list for system-collections so should that one then be added to that list aswell?